src/uct/ib/rdmacm/rdmacm_cm_ep.c

```diff
@@ -404,6 +404,7 @@ static ucs_status_t uct_rdamcm_cm_ep_client_init(uct_rdmacm_cm_ep_t *cep,
     char dst_ip_port_str[UCS_SOCKADDR_STRING_LEN];
     char ep_str[UCT_RDMACM_EP_STRING_LEN];
     ucs_status_t status;
+    struct sockaddr *src_addr;
```

Comment:
`const`

Reply:
this can't be changed to const as following rdma_resolve_addr need non const

Reply:
make it const, and cast to (struct sockaddr *) when passing to rdma_resolve_addr()

Reply:
changed accordingly